make install woes

I have some weird problems with the Makefile - make install fails to install /usr/local/bin/ocrd-olena-binarize:

~~~
% sudo make install PREFIX=/usr/local
test -x /usr/local/bin/scribo-cli && \
/usr/local/bin/scribo-cli sauvola --help >/dev/null 2>&1 || \
	make build-olena
which ocrd >/dev/null 2>&1 || \
	/usr/bin/pip3 install ocrd # needed for ocrd CLI (and bashlib)
~~~

I read the Makefile that `install` depends on `/usr/local/bin/ocrd-olena-binarize` existing but it doesn't:

~~~
% ls -l /usr/local/bin/ocrd-olena-binarize
ls: cannot access '/usr/local/bin/ocrd-olena-binarize': No such file or directory
~~~

More weirdly, without sudo, it tries to install:

~~~
% make install PREFIX=/usr/local
test -x /usr/local/bin/scribo-cli && \
/usr/local/bin/scribo-cli sauvola --help >/dev/null 2>&1 || \
	make build-olena
which ocrd >/dev/null 2>&1 || \
	/usr/bin/pip3 install ocrd # needed for ocrd CLI (and bashlib)
sed 's|^SHAREDIR=.*|SHAREDIR="/usr/local/share/ocrd_olena"|;s|^PYTHON=.*|PYTHON="/usr/bin/python3"|' ocrd-olena-binarize > /usr/local/bin/ocrd-olena-binarize
/bin/sh: /usr/local/bin/ocrd-olena-binarize: Permission denied
make: *** [Makefile:82: /usr/local/bin/ocrd-olena-binarize] Error 1
~~~

--- a/Makefile
+++ b/Makefile
@@ -65,7 +65,7 @@ deps: #deps-ubuntu
 	test -x $(BINDIR)/scribo-cli && \
 	$(BINDIR)/scribo-cli sauvola --help >/dev/null 2>&1 || \
 		$(MAKE) build-olena
-	which ocrd >/dev/null 2>&1 || \
+	ocrd ocrd-tool --help >/dev/null 2>&1 || \
 		$(PIP) install ocrd # needed for ocrd CLI (and bashlib)
 
 # Install

